### ISSUE TITLE
Alert: reply expired - accessibilityHint is missing on Compose a message button

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#25915]
## What happened?
Both iOS and Android seem to be missing the accessibilityHint for the Compose a message button.

## Specs:
- Device: iOS and Android w/screen reader enabled

## Steps to Reproduce
- Navigate to an old message (>45 days old) and view the notice.
- Read notice as well as button with screen reader.

## Desired behavior
Compose a message button:

- accessibilityLabel= "compose a message"
- accessibilityRole= "button"
- accessibilityHint= "go to the compose a message page"

Screen reader will read the accessibilityHint.

## Acceptance Criteria
See above.

## Bug Severity
- 4 - low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/22617

## Screen shot(s) and additional information
n/a

**Comment by digitalken** [comment]
Since original story is launch critical, marking as a sev 3 bug.

**Comment by digitalken** [comment]
PASS 6/17/2021 - Sounds good on both iOS and Android.  Closing.
